pages/borrow/index.tsx

```diff
@@ -1017,54 +1018,66 @@ const Markets: NextPage = () => {
   async function executeRepay(val: any, toast: ToastProps['toast']) {
     try {
       if (!val) return toast.error('Please provide a value');
+      // add additional value if user wants to repay 100% of loan due to interest rate not being included
+      if (val == userDebt) val += 1;
```

Comment:
@bowtiedpyro update to value used in SSR branch as discussed
make sure that all related updates regarding the SDK bump are included in this PR/branch 
(no additional ones in the SSR branch)

Reply:
done